jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/JettyEmbedder.java

```diff
@@ -245,15 +245,14 @@ protected void join() throws InterruptedException
      */
     private void configure() throws Exception
     {
-        /* Configure the server */
-        //apply any configs from jetty.xml files first 
-        Server tmp = ServerSupport.applyXmlConfigurations(server, jettyXmlFiles, jettyProperties);
-        if (server == null)
-            server = tmp;
 
         if (server == null)
             server = new Server();
-        
+
+        /* Configure the server */
+        //apply any configs from jetty.xml files first
+        Server tmp = ServerSupport.applyXmlConfigurations(server, jettyXmlFiles, jettyProperties);
```

Comment:
What do you do then with that `tmp` server?

Reply:
Fixed